### PR TITLE
Append missing loongson 2e/2f entries in mips-dis.c and mybfd.h

### PR DESCRIPTION
**Pull request by RinGotou**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Just add some missing entries inside mybfd.h and mips-dis.c from latest source code of binutils, for loongson MIPS processor support. 

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by wargio** [comment]
can you add tests on this?

**Comment by RinGotou** [comment]
> can you add tests on this?

I'm not familiar with the test suite for now... maybe I will add some tests after deeper reading of codes.

**Comment by XVilka** [comment]
cc @junchao-loongson

File: libr/asm/arch/mips/gnu/mips-dis.c [hunk 1]
```diff
@@ -430,6 +430,16 @@ const struct mips_arch_choice mips_arch_choices[] =
     mips_cp0sel_names_sb1, ARRAY_SIZE (mips_cp0sel_names_sb1),
     mips_hwr_names_numeric },
 
+  { "loongson2e", 1, bfd_mach_mips_loongson_2e, CPU_LOONGSON_2E,
```
Comment (trufae) [inline libr/asm/arch/mips/gnu/mips-dis.c:433]: this is not used by asm.cpu, right? i mean, the pr only adds the definitions for those mips archs. so it's good to go, but its unused code iiuic

**Pull request merged**